packages/asc-ui/src/components/Breadcrumbs/BreadcrumbsItem/BreadcrumbsItemStyle.ts

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components'
+import { themeColor, themeSpacing } from '../../../utils/themeUtils'
+import Icon from '../../Icon'
+
+export interface liProps {
```

Comment:
When defining an interface make sure to capitalize the first letter as well. In this case this interface is only used in `BreadcrumbsItem` so I would define it there as `BreadcrumbsItemProps`.

Reply:
done that

Reply:
if I change the liProps to Liprops the unit tests fail:  packages/asc-ui/src/components/Breadcrumbs/BreadcrumbsItem/BreadcrumbsItem.tsx:4:3 - error TS2724: Module '"./BreadcrumbsItemStyle"' has no exported member 'LiProps'. Did you mean 'liProps'?

Reply:
but I will remove the BreadcrumbsItem anyway: because that is the other thing